addons/website_sale_collect/models/stock_warehouse.py

```diff
@@ -2,10 +2,47 @@
 
 from odoo import fields, models
 
+from odoo.tools.misc import format_duration
+
 
 class StockWarehouse(models.Model):
     _inherit = 'stock.warehouse'
 
     opening_hours = fields.Many2one(
         string="Opening Hours", comodel_name='resource.calendar', check_company=True
     )
+
+    def _format_pickup_location_values(self):
```

Comment:
```suggestion
    def _prepare_pickup_location_data(self):
```
or `_values` if this is different from what we usually call location data.